joomla/joomla-cms
# [com_search] Use page title from menu item.

Pull Request for Issue # .

### Summary of Changes
This corrects page title in search view.

### Testing Instructions
Create menu item for 'Search » Search Form or Search Results'. Check the page title when:
a) 'Page Title' in menu item is not set.
b) 'Page Title' in menu item is set to 0.
c) 'Page Title' in menu item is set to anything other than 0.
d) When accessing component directly

### Expected result
a) Page title is the same as menu item title.
b) Page title is 0.
c) Page title is whatever entered in 'Page Title' field.
d) Page title is 'Search'.

### Actual result
a) Page title is 'Search' (incorrect).
b) Page title is 'Search' (incorrect).
c) Page title is whatever entered in 'Page Title' field (correct).
d) Page title is 'Search' (correct).

### Documentation Changes Required
No.
